settings.gradle

```diff
@@ -113,6 +113,7 @@ include(':ls-extensions:performance-analyzer-services')
 include(':ls-extensions:partial-parser')
 include(':ls-extensions:trigger-service')
 include(':ls-extensions:bal-shell-service')
+include(':ls-extensions:sa-modeling-service')
```

Comment:
Shall we update the module name into something like `project-design-service`/`project-modeling-service` as per the recent name related discussions we had?

Reply:
Updated to `project-design-service`